modules/local/collect_featurecounts.nf

```diff
@@ -0,0 +1,73 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+
+process COLLECT_FEATURECOUNTS {
+    tag "counts${options.suffix}.tsv.gz"
+    label 'process_high'
+
+    conda (params.enable_conda ? "conda-forge::r-tidyverse=1.3.1 conda-forge::r-data.table=1.14.0 conda-forge::r-dtplyr=1.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-508c9bc5e929a77a9708902b1deca248c0c84689:0bb5bee2557136d28549f41d3faa08485e967aa1-0' :
+        'quay.io/biocontainers/mulled-v2-508c9bc5e929a77a9708902b1deca248c0c84689:0bb5bee2557136d28549f41d3faa08485e967aa1-0' }"
+
+    input:
+        path inputfiles
+
+    output:
+        path "*.tsv.gz"    , emit: counts
+        path "versions.yml", emit: versions
+
+    script:
+    def software = getSoftwareName(task.process)
+    def args     = task.ext.args ?: ''
+    
```

Comment:
This row contains trailing white spaces, according to github actions (EditorConfig).